doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,550 @@
+
+* :ref:`lib_dfu_target`:
+
+  * Added the configuration option :option:`CONFIG_DFU_TARGET_MCUBOOT_SAVE_PROGRESS`, which uses Zephyr's :ref:`zephyr:settings` subsystem.
+    When this option is enabled, the write progress of an MCUboot style upgrade is stored, so that the progress is retained when the device reboots.
+  * Fixed a bug where :cpp:func:`dfu_target_done` logged the error message ``unable to deinitialize dfu resource`` when no target was initialized.
+
+* Moved the following libraries from ``drivers/`` to ``lib/``:
+
+  * :ref:`fprotect_readme`
+  * :ref:`st25r3911b_nfc_readme`
+  * ``adp536x``
+  * ``flash_patch``
+
+
+Crypto
+======
+
+* Added low-level cryptographic test suite using NIST, RFCs, and custom test vectors.
+* :ref:`nrf_cc310_mbedcrypto_readme`/:ref:`nrf_cc310_platform_readme` v0.9.2:
+
+  * Fixed power-efficiency issues.
+  * Added experimental use of CryptoCell interrupt instead of busy-waits.
+* :ref:`lib_hw_cc310`:
+
+  * Added support for CryptoCell interrupt.
+
+nRF Bluetooth LE Controller
+===========================
+
+* Updated the :ref:`nrfxlib:ble_controller` libraries:
+
+  * Removed version numbers for the libraries.
+  * Added preliminary support for the S140 variant with the nRF5340 device.
+    The Bluetooth LE Controller for nRF5340 supports the same feature set as its nRF52 Series counterpart.
+  * Moved some APIs to :ref:`nrfxlib:mpsl`.
+    The library must now be linked together with MPSL.
+  * Made Data Length Extensions a configurable feature.
+  * Fixed an issue where an assert could occur when receiving a packet with a CRC error after performing a data length procedure on Coded PHY.
+
+  For details, see the :ref:`nrfxlib:ble_controller_changelog`.
+
+Multi-Protocol Service Layer (MPSL)
+===================================
+
+* Updated the :ref:`nrfxlib:mpsl` libraries:
+
+  * Removed version numbers for the libraries.
+  * Added a library version with preliminary support for the nRF5340 device.
+    The feature set is the same as in the MPSL library for nRF52.
+
+  For details, see the :ref:`nrfxlib:mpsl_changelog`.
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* Added the following sample:
+
+  * :ref:`peripheral_gatt_dm` - demonstrates how to use the :ref:`gatt_dm_readme`.
+
+* :ref:`nrf_bt_scan_readme`:
+
+  * Added an option to update the initial connection parameters.
+
+* :ref:`nus_c_readme`, :ref:`bas_c_readme`, and :ref:`dfu_smp_c_readme`:
+
+  * Fixed an issue where it was not possible to subscribe to the service notifications more than once.
+
+* Updated the :ref:`central_uart` sample to handle data packets that are longer than 212 bytes.
+  Enabled UART flow control to avoid data loss.
+
+* Enabled UART flow control in the :ref:`peripheral_uart` sample to avoid data loss.
+
+* Fixed the :ref:`ble_throughput` sample, which did not achieve the expected throughput measurements.
```

Comment:
HID Service: use `bt_gatt_is_subscribed()` instead of an internal implementation in the HID service

Reply:
> HID Service: use `bt_gatt_is_subscribed()` instead of an internal implementation in the HID service

This doesn't affect the user I think. Only an internal change in the service.

Reply:
Not added for now.